### PR TITLE
PR: Algo shellsort in java

### PR DESCRIPTION
**Pull request by sowmyasri-Z**
This PR is about shell sort in java

### CODE REVIEW

**Comment by sowmyasri-Z** [comment]
can u please check my work..

**Comment by Abishethvarman** [comment]
> _ No description provided. _

provide description

File: J-shellsort/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# 🐚 SHELL SORT 🐚
+# DEFINITION 🥇
```
Comment (Abishethvarman) [inline J-shellsort/README.md:2]: make a gap between the headings

**Review by Abishethvarman — CHANGES_REQUESTED** [review]
Content is ok

**Comment by sowmyasri-Z** [comment]
requested changes are done...

File: J-shellsort/README.md
Comment (Abishethvarman) [inline J-shellsort/README.md:1]: check the file location. Place it in the correct path

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

File: J-shellsort/README.md
Comment (sowmyasri-Z) [reply J-shellsort/README.md:1]: sorry i did not get that. Can u give more details on the issue

Comment (Abishethvarman) [reply J-shellsort/README.md:1]: image
 your contribution has added here. But it should be added under the java folder under the algorithm 
image

**Comment by Abishethvarman** [comment]
This PR closed due to mislocation issue

**Pull request closed without merging**